contracts/truefi/CurvePool.sol

```diff
@@ -41,13 +44,24 @@ contract CurvePool is ITruePool, ERC20, ReentrancyGuard, Ownable {
         return _currencyToken;
     }
 
+    function totalLiquidityTokenBalance() public view returns (uint256) {
+        return _curvePool.token().balanceOf(address(this)).add(_curveGauge.balanceOf(address(this)));
+    }
+
     function poolValue() public view returns (uint256) {
         return
             _currencyToken.balanceOf(address(this)).add(_lender.value()).add(
-                _curvePool.token().balanceOf(address(this)).mul(_curvePool.curve().get_virtual_price()).div(1 ether)
+                totalLiquidityTokenBalance().mul(_curvePool.curve().get_virtual_price()).div(1 ether)
             );
     }
 
+    function prepareTokens(uint256 neededAmount) internal {
```

Comment:
I'd rename it. But don't know how yet

Reply:
Smth like `restock`, `refuel` maybe?

Reply:
Renamed

Reply:
Oh. Hmmm... I was thinking about some longer name as this is not an external method